dll/cpl/input/input_list.c

```diff
@@ -323,14 +323,56 @@ InputList_AddInputMethodToUserRegistry(DWORD dwIndex, INPUT_LIST_NODE *pNode)
         }
     }
 
+    /* 
+     * First time more then 1 input language exists switching hotkeys needed
+     * set default values for language/layout switching hotkeys
+     */
+    if (2 == dwIndex) {
```

Comment:
Please let's keep existing (and if possible, sane [code style](https://reactos.org/wiki/Coding_Style) ):
```suggestion
    if (dwIndex == 2)
    {
```

Reply:
Done